### ISSUE TITLE
Degree of decentralization

### ISSUE THREAD

**Issue by lrettig** [spacemeshos/api#22]
I think this can be calculated outside the API (e.g., by the dash/explorer backend) on the basis of data it already has access to from the API, using the math in https://github.com/spacemeshos/product/blob/129ef13a76d91a8f92ba079e9303697290596337/dashboard.md. See also https://github.com/spacemeshos/product/pull/77.

**Comment by lrettig** [comment]
This probably will not result in a new rpc endpoint but we can keep this here as a placeholder until this has been implemented in the dash-backend and node-mock. CC @primaveralina

**Comment by avive** [comment]
yes, I agree. Maybe this issue should be moved to the explore-backend repo?

**Comment by lrettig** [comment]
Let's discuss this with @primaveralina and team tomorrow and decide where it should live then. I tend to agree, I don't think we need a new endpoint for this.
